projects/demo/src/modules/components/tab-bar/index.html

```diff
@@ -44,58 +45,28 @@
                 <code>routerLinkActive</code>
                 directive.
             </tui-notification>
-            <tui-tab-bar-example-2 class="bar" />
         </tui-doc-example>
 
         <tui-doc-example
             id="customization"
             heading="Customization"
-            [content]="example3"
-        >
-            <tui-tab-bar-example-3 class="bar" />
-        </tui-doc-example>
+            [component]="3 | tuiComponent"
+            [content]="3 | tuiExample: 'html,ts'"
+        />
 
         <tui-doc-example
             id="skeleton"
             heading="Skeleton"
-            [content]="example4"
+            [component]="4 | tuiComponent"
+            [content]="4 | tuiExample: 'html,ts'"
         >
             <tui-notification class="bar tui-space_bottom-4">
                 When there are no
                 <code>TabBarItem</code>
                 children, the component shows skeleton for 4 items
             </tui-notification>
-            <tui-tab-bar-example-4 class="bar" />
         </tui-doc-example>
     </ng-template>
 
-    <ng-template pageTab="Setup">
-        <ol class="b-demo-steps">
-            <li>
-                <p>
-                    Import
-                    <code>TuiTabBarModule</code>
-                    into your main module
-                </p>
-
-                <tui-doc-code
-                    filename="my.module.ts"
-                    [code]="exampleModule"
-                />
-            </li>
-
-            <li>
-                <p>
-                    Add to the template inside
-                    <code>Root</code>
-                    component at the required layer and position with CSS:
-                </p>
```

Comment:
I'm not sure that basic `<tui-setup />` is enough here 🤔

Reply:
I add that comment inside template

<img width="550" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/e1fc0525-6519-4356-a6bb-22787caa2103">